exporter/awscloudwatchlogsexporter/exporter.go

```diff
@@ -106,18 +107,30 @@ func (e *exporter) ConsumeLogs(ctx context.Context, ld plog.Logs) error {
 			InputLogEvent: logEvent,
 			GeneratedTime: time.Now(),
 		}
+		body := &cwLogBody{}
+		err := json.Unmarshal([]byte(*logEvent.InputLogEvent.Message), body)
+		if err != nil {
+			return err
```

Comment:
Should a message failing to marshal force the entire function to return an error without completion?

Reply:
good catch thanks!